invenio_app_ils/documents/jsonresolvers/document_circulation.py

```diff
@@ -82,8 +83,16 @@ def circulation_resolver(document_pid):
             if bucket["key"] in "FOR_REFERENCE_ONLY":
                 items_for_reference_count = bucket["doc_count"]
 
+        active_ill_loans = (
+            BorrowingRequestsSearch()
+            .get_active_ill_loans_for_document(document_pid)
+            .count()
+        )
```

Comment:
Looks good to me :+1:

I was wondering if it possible to avoid an extra search, but it might not be reliable. Given an active loan, one of the only way we can know if it's an ILL loan is to check the metadata's `item_pid.type`: in case of an ILS item it would be `pitmid`, and otherwise `illbid`. So the idea would be to filter out the loans by their item before aggregating them, so we are sure we are dealing with loans on ILS items. (if a loan has no item we should not filter it out)